lbrynet/daemon/Daemon.py

```diff
@@ -2389,12 +2389,13 @@ def jsonrpc_send_amount_to_address(self, amount, address):
         elif not amount:
             raise NullFundsError()
```

Comment:
This is wrong, if amount is `None` then the `amount < 0` will throw an exception. First there need to be a check that amount is not `None` and then a comparison can be done.

```python
>>> None < 0
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: '<' not supported between instances of 'NoneType' and 'int'
```